src/client/src/services/credit/creditRfqResponses/creditRfqResponses.ts

```diff
@@ -64,55 +60,6 @@ const dealersResponses$ = createdCreditRfq$.pipe(
   ),
 )
 
-const sellSideTicket$ = createdCreditRfq$.pipe(
-  withLatestFrom(creditDealers$),
-  map(
-    ([
-      {
-        rfqId,
-        request: { dealerIds },
-      },
-      creditDealers,
-    ]: [CreatedCreditRfq, DealerBody[]]) => {
-      const adaptiveBankId = creditDealers.find(
-        (dealer) => dealer.name === ADAPTIVE_BANK_NAME,
-      )?.id
-      const dealerId = dealerIds.find((id) => id === adaptiveBankId)
-      return dealerId !== undefined
-        ? {
-            rfqId,
-            dealerId,
-          }
-        : undefined
-    },
-  ),
-  tap((ticketToOpen) => {
-    if (ticketToOpen) {
-      openDealerTicketForRfq(
-        ticketToOpen.rfqId.toString(),
-        ticketToOpen.dealerId.toString(),
-      )
-    }
-  }),
-)
-
-// Open the sell side ticket to manually respond to RFQs
-function openDealerTicketForRfq(rfqId: string, dealerId: string) {
-  openWindow({
-    url: constructUrl(
-      ROUTES_CONFIG.sellSideTicket
-        .replace(":rfqId", rfqId)
-        .replace(":dealerId", dealerId),
-    ),
-    name: `CreditRFQ-${rfqId}-${dealerId}`,
-    displayName: "Credit RFQ",
-    width: 330,
-    height: CREDIT_SELL_SIDE_TICKET_HEIGHT,
-    x: window.innerWidth - 330,
-    y: window.innerHeight - CREDIT_SELL_SIDE_TICKET_HEIGHT,
-  })
-}
-
 function sendRandomQuoteAfterDelay(
```

Comment:
this is a bit of a surprise .. I thought the server-side was doing this ...

Reply:
Yeah...